hash/spectrum_cass.xml

```diff
@@ -163202,114 +163663,179 @@ Tape 2: Cell of the Ridges
 
+	<software name="gamestapii">
+		<description>Games Tape II (Pi Software)</description>
+		<year>1983</year>
+		<publisher>Pi Software</publisher>
+		<notes><![CDATA[Compilation contents:
+Side A - Game 1: Blitz
+Side A - Game 2: Snake Byte
+Side A - Game 3: Fighter
+]]></notes>
 
-	<software name="gamestap_k" cloneof="gamestap">
-		<description>Games Tape I (Riko Data)</description>
-		<year>19??</year>
-		<publisher>&lt;unknown&gt;</publisher>
+		<info name="alt_title" value="Games Tape Two"/>
 		<part name="cass" interface="spectrum_cass">
-			<dataarea name="cass" size="22797">
-				<rom name="games tape i (riko data).tzx" size="22797" crc="b3ffb283" sha1="3aa8f9f659813b487d4414787603d605e7690e5a"/>
+			<dataarea name="cass" size="18484">
+				<rom name="games tape 2 (pi software).tzx" size="18484" crc="ab3d3ba4" sha1="b37123d8472fb8317788b9ee9e2122f62c323692"/>
 			</dataarea>
 		</part>
 	</software>
 
-	<software name="gamestap_l" cloneof="gamestap">
+	<software name="gamestapii_a" cloneof="gamestapii">
 		<description>Games Tape II (Riko Data)</description>
-		<year>19??</year>
-		<publisher>&lt;unknown&gt;</publisher>
+		<year>1983</year>
+		<publisher>Riko Data</publisher>
 		<part name="cass" interface="spectrum_cass">
 			<dataarea name="cass" size="18478">
 				<rom name="games tape ii (riko data).tzx" size="18478" crc="26701b8b" sha1="4dc86d69bc1392957ef253e069f5af70856bee64"/>
 			</dataarea>
 		</part>
 	</software>
 
-	<software name="gamestap_f" cloneof="gamestap">
-		<description>Games Tape 2 (Pi Software)</description>
-		<year>19??</year>
-		<publisher>&lt;unknown&gt;</publisher>
-		<part name="cass" interface="spectrum_cass">
-			<dataarea name="cass" size="18484">
-				<rom name="games tape 2 (pi software).tzx" size="18484" crc="ab3d3ba4" sha1="b37123d8472fb8317788b9ee9e2122f62c323692"/>
+	<software name="gamestap3">
+		<description>Games Tape 3 (Gilsoft, release 1)</description>
+		<year>1983</year>
+		<publisher>Gilsoft International</publisher>
+		<notes><![CDATA[Compilation contents:
+Side A: Time-Line
+Side B: Tasks
+]]></notes>
+
+		<part name="cass1" interface="spectrum_cass">
+			<feature name="part_id" value="Time-Line"/>
+			<dataarea name="cass" size="17403">
+				<rom name="games tape 3 (gilsoft) - release 1 - side 1.tzx" size="17403" crc="0c645102" sha1="a944b6cbf4306ed6955a5af87ccaceb21f9fc54e"/>
+			</dataarea>
+		</part>
+		<part name="cass2" interface="spectrum_cass">
+			<feature name="part_id" value="Tasks"/>
+			<dataarea name="cass" size="16981">
+				<rom name="games tape 3 (gilsoft) - release 1 - side 2.tzx" size="16981" crc="5d597d9d" sha1="a07cdc029e2964511b0250223c3e51971a53bf8f"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="gamestap3_a" cloneof="gamestap3">
+		<description>Games Tape 3 (Gilsoft, release 2)</description>
+		<year>1983</year>
+		<publisher>Gilsoft International</publisher>
+		<notes><![CDATA[Compilation contents:
+Side A: Time-Line
+Side B: Tasks
+]]></notes>
```

Comment:
Does this really add value when the information is present on the label in the `part_id` for each side anyway?  Same for other items where all the information is already present and parseable.